docs/relnotes.md

```diff
@@ -62,7 +62,8 @@ Published on TO_BE_DEFINED.
 
 #### Features
 
-   - TODO alternative storage system
+   - New repository: SQLLite can now be used as a replacement for filesystem as the taipy repository.
```

Comment:
@jrobinAV @FabienLelaquais Any feedback on the text is apreciated. Cheers.

Reply:
I would group both entries, and I would not use the term repository which can be misinterpreted.

Reply:
And I would capitalize 'Taipy'